src/includes/performance/connect-errors-spans/java.mdx

```diff
@@ -19,7 +19,6 @@ import io.sentry.Sentry;
 import io.sentry.ISpan;
 
 ITransaction span = Sentry.startTransaction(item.getTransactionName(), "task");
-Sentry.configureScope(scope -> scope.setTransaction(span));
```

Comment:
I think this would require also a note to the migration page right

Reply:
Added migration docs.